config/install/fvm.settings.yml

```diff
@@ -34,3 +35,25 @@ paragraph__embed__view_modes:
 paragraph__hero_media: 0
 paragraph__hero_media__view_modes:
   preview: 0
+block_content__cards: 1
+block_content__cards__view_modes:
+  cards_grid_3xn_: cards_grid_3xn_
+  preview: preview
+media__video: 0
+media__video__view_modes:
+  gallery: 0
+node__smart_article: 0
+node__smart_article__view_modes:
+  teaser: 0
+paragraph__form: 0
+paragraph__form__view_modes:
+  preview: 0
+paragraph__gallery: 0
+paragraph__gallery__view_modes:
+  preview: 0
+paragraph__gallery_carousel: 0
+paragraph__gallery_carousel__view_modes:
+  preview: 0
+paragraph__view: 0
+paragraph__view__view_modes:
+  preview: 0
```

Comment:
I believe, this is incorrect. there are entities in the config which are not installed on default profile install, like smart article. Can you please see.

Reply:
Addressed